suite-native/device/src/utils.ts

```diff
@@ -42,7 +38,7 @@ export const isDeviceEventAction = <T extends DeviceEvent['type']>(
     actionType: T,
 ): action is { type: T; payload: Device } => action.type === actionType;
 
-export const isDeviceSetupSupported = (model: DeviceModelInternal) => {
+export const getIsDeviceSetupSupported = (model: DeviceModelInternal) => {
```

Comment:
I'd keep the original name if possible to stay consistent with other functions in this file.
```suggestion
export const isDeviceSetupSupported = (model: DeviceModelInternal) => {
```

Reply:
I'd actually prefer to prefix functions with `getSomething` in general. Most functions in common are prefixed this way so we don't have a repository consistency sadly so if you don't have a strong opinion on this, I'd prefer to keep this.